src/js/components/search/visualizations/rank/RankVisualizationSection.jsx

```diff
@@ -73,15 +73,15 @@ export default class RankVisualizationSection extends React.Component {
         let disablePrev = false;
         let hidePager = '';
 
-        if (this.props.total < this.props.page + 1) {
+        if (!this.props.hasNextPage) {
             disableNext = true;
         }
 
-        if (this.props.page <= 1) {
+        if (!this.props.hasPreviousPage) {
             disablePrev = true;
         }
 
-        if (this.props.total < 1 || this.props.loading) {
+        if ((!this.props.hasPreviousPage && !this.props.hasNextPage) || this.props.loading) {
```

Comment:
For readability, if we make the above changes to `disableNext` and `disablePrev`, this statement can now become `if ((disableNext && disablePrev) || this.props.loading)`